.github/workflows/build.yml

```diff
@@ -107,48 +108,48 @@ jobs:
     - name: Set extra CMake flags
       run: |
         echo "CMAKE_OPTIONS=${CMAKE_OPTIONS} -DENABLE_DYNAMIC_LOADING=true" >> $GITHUB_ENV
+        if [[ "${{ matrix.mpi }}" == "openmpi" || "${{ matrix.mpi }}" == "mpich" ]]; then
+          echo "CMAKE_OPTIONS=${CMAKE_OPTIONS} -DTEST_MPI_PROCS=2" >> $GITHUB_ENV
+        fi
 
     - name: Configure build
       run: >-
-        cmake -B _build -G Ninja
-        -DCMAKE_INSTALL_PREFIX=${PWD}/_install
+        cmake -B ${BUILD_DIR} -G Ninja
+        -DCMAKE_INSTALL_PREFIX=${PWD}/${INSTALL_DIR}
         -DCMAKE_BUILD_TYPE=${{ matrix.config }}
         ${CMAKE_OPTIONS}
         -DWITH_MPI=${WITH_MPI}
 
     - name: Build project
-      run: cmake --build ${BUILD_DIR}
+      run: cmake --build ${BUILD_DIR} -j 2
```

Comment:
I'm wondering if just "-j" without number would also be OK, as the VM should be limited to 2 processes (and if it's ever changed to run on something larger, will also take advantage).

Reply:
I guess that depends on whether they've enabled hyperthreading. In this case the hard-coded -j 2 will (?) restrict the compilation to the two physical cores.

Reply:
Seems that I'm wrong in any case, see:
https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories
They seem to have upgraded to 4 cores for public repos.

Reply:
I have changed things accordingly...